OpenAPI/kiota/install.md

```diff
@@ -63,6 +63,12 @@ Execute the following command to install the tool.
 dotnet tool install --global Microsoft.OpenApi.Kiota
 ```
 
+## Install with asdf
```

Comment:
can you also add an entry to the table at line 12 please?

Reply:
done 👍